src/main/org/apache/tools/ant/ComponentHelper.java

```diff
@@ -452,12 +452,7 @@ public void checkTaskClass(final Class<?> taskClass) throws BuildException {
     }
 
     /**
-     * Adds a new datatype definition.
-     * Attempting to override an existing definition with an
-     * equivalent one (i.e. with the same classname) results in
-     * a verbose log message. Attempting to override an existing definition
-     * with a different one results in a warning log message, but the
-     * definition is changed.
+     * Update the component definition table with a new or modified definition.
      *
```

Comment:
This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?